app/models/service.rb

```diff
@@ -11,6 +12,15 @@ class Service < ApplicationRecord
     "Do Nothing" => nil
   }.freeze
 
+  POWER_STATE = {
+    "on"         => :on,
+    "off"        => :off,
+    "on_partial" => :on_partial,
+    "suspended"  => :on_partial,
+    "starting"   => :starting,
+    "stopping"   => :stopping
+  }.freeze
```

Comment:
Can we get away without needing this `POWER_STATE` hash?  Only one that is different is `suspended` and setting that to `on_partial` seems inaccurate.